core/app/models/BaseViewModel.java

```diff
@@ -0,0 +1,7 @@
+package models;
+
+public class BaseViewModel {
+
+    public String user;
```

Comment:
Do you think we should have constructors for the `ViewModel` classes? I feel like not having them could cause some minor problems for people working on the project in the future. But again, I'm not sure about the conventions of the view model/ scala templating stuff so it's fine if this is how it's typically done.